common/txmgr/txmgr.go

```diff
@@ -191,10 +191,12 @@ func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) Start(ctx
 			return fmt.Errorf("Txm: Estimator failed to start: %w", err)
 		}
 
+		b.logger.Infow("Txm starting tracker")
```

Comment:
```suggestion
		b.logger.Info("Txm starting tracker")
```
Or maybe drop to debug too?
```suggestion
		b.logger.Debug("Txm starting tracker")
```